packit_service/worker/fedmsg_handlers.py

```diff
@@ -141,6 +141,20 @@ def __init__(self, config: ServiceConfig, job: JobConfig, event: CoprBuildEvent)
         self.project = self.event.get_project()
         self.package_config = self.event.get_package_config()
 
+    def was_last_build_successful(self):
+        """
+        Check if the last copr build of the PR was succesful
+        :return: bool
+        """
+        comments = self.project.get_pr_comments(pr_id=self.event.pr_id, reverse=True)
+        for comment in comments:
+            if comment.author.startswith("packit-as-a-service"):
```

Comment:
Nice proposal for extending ogr API: https://github.com/packit-service/ogr/issues/240